components/team-member/__tests__/TeamMember.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import TeamMember from '../TeamMember';
+
+const teamMembers = {
```

Comment:
I would rename it to teamMember, teamMembers suggest that the type of data is an Array.